inst/schema/SurveyFilters.schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "sex": {
+      "type": "array",
+      "items": { "type": "string" }
+    },
+    "age": {
+      "type": "array",
+      "items": { "type": "string" }
+    },
+    "region": {
+      "type": "array",
+      "items": {
+        "$ref": "NestedFilterOption.schema.json"
```

Comment:
Will these differ between survey, program and ANC? Or can we just have a master list of regions for a country (if some regions are missing for a particular data set they just appear as grey/no data in the leaflet map)

Reply:
Yeah, I was wondering that too, this is potentially quite a bulky data structure to pass around multiple times, just on the offchance that one might contain a slightly reduced list. We're expecting the norm to be that all files have data from all areas (?). So if we did just have one master list, maybe it should be the shape file validate that returns that?

Reply:
Yeah, I feel like returning this with the shapefile would make sense. In the case of missing data regions will just show as greyed out on the map, can have a legend to indicate that means "no data".